src/signal-handling.c

```diff
@@ -88,12 +93,134 @@ JL_DLLEXPORT void jl_exit_on_sigint(int on)
     exit_on_sigint = on;
 }
 
+void jl_show_sigill(void *_ctx);
+
 #if defined(_WIN32)
 #include "signals-win.c"
 #else
 #include "signals-unix.c"
 #endif
 
+void jl_show_sigill(void *_ctx)
+{
+#if defined(_CPU_X86_64_) || defined(_CPU_X86_)
+    uintptr_t pc = 0;
+#  if defined(_OS_LINUX_) && defined(_CPU_X86_64_)
```

Comment:
The indentation in the preprocessor logic here is weird and seems to be inconsistent with other files (e.g. debuginfo.c), which don't use indentation on nested conditions.